index.html

```diff
@@ -175,7 +178,7 @@
         defines the following enumeration:
       </p>
       <pre class="idl">
-        enum WakeLockType { "screen", "system" };
+        [SecureContext] enum WakeLockType { "screen", "system" };
```

Comment:
`[SecureContext]` [doesn't apply to enums](https://heycam.github.io/webidl/#SecureContext).